gitbook/src/fundamentals/sync-modes.md

```diff
@@ -0,0 +1,40 @@
+# Sync Modes
+
+Erigon 3 introduces a flexible approach to node configuration, offering three distinct types to suit various user needs. Depending on your need, you can choose from three different node types.
+
+| Usage                 | Minimal Node | Full Node | Archive Node |
+| --------------------- | ------------ | --------- | ------------ |
+| Privacy, RPC          | **Yes**      | **Yes**   | **Yes**      |
+| Contribute to network | No           | **Yes**   | **Yes**      |
+| Research              | No           | No        | **Yes**      |
+| Staking               | **Yes**      | **Yes**   | **Yes**      |
+
+By **default**, Erigon run as a [full node](sync-modes.md#full-node).
+
+In order to switch type of node, you must first delete the `/chaindata` folder in the chosen `--datadir` directory and re-sync from scratch.
+
+{% hint style="info" %}
+**Persisting receipts**, which are pre-calculated receipts, are enabled by default for minimal and full node. To disable them use the flag `--persist.receipts=false`.
```

Comment:
use "persisting receipts" if need increase requests-per-second (latency and throughput) on Receipts/Logs-related RPC's